Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM python:2.7.12
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+                    build-essential \
```

Comment:
The thing I mentioned in the other PR about removing build deps can apply here. :)